### PR TITLE
Link to a list of configurable lints in documentation

### PR DESCRIPTION
**Pull request by Alexendoo**
changelog: none

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -197,8 +197,8 @@ disallowed-names = ["toto", "tata", "titi"]
 cognitive-complexity-threshold = 30
 ```
 
-See the [list of lints](https://rust-lang.github.io/rust-clippy/master/index.html) for more information about which
-lints can be configured and the meaning of the variables.
+See the [list of configurable lints](https://rust-lang.github.io/rust-clippy/master/index.html#Configuration)
+for more information about the meaning of the variables.
```
Comment (xFrednet) [inline README.md:201]: Do we maybe want to include a comment that they have to open the lint descriptions to see the configs?

```suggestion
open the lint description for more information about the meaning of these variables.
```

Comment (Alexendoo) [reply README.md:201]: I'm struggling to come up with a phrasing that doesn't sound awkward

Comment (xFrednet) [reply README.md:201]: What about a simple: "See the _list of configurable lints_, the lint description contains the name and effect of these variables"

We could also try an alternative and add a new part to the list/book, which lists all config variable as first class citizens. This is less related to this change and more a general idea

**Review by xFrednet — APPROVED** [review]

**Comment by xFrednet** [comment]
LGTM, thank you, let's hope that it helps a bit :)

@bors r+

**Pull request merged**